packages/driver/test/cypress/integration/commands/actions/click_spec.js

```diff
@@ -1470,14 +1470,30 @@ describe('src/cy/commands/actions/click', () => {
 
         cy.on('fail', (err) => {
           expect(clicked).to.eq(1)
-          expect(err.message).to.include('cy.click() failed because this element')
+          expect(err.message).to.include('cy.click() failed because this element is detached from the DOM')
 
           done()
         })
 
         cy.get(':checkbox:first').click().click()
       })
 
+      it('throws when subject is detached during actionability', (done) => {
+        cy.on('fail', (err) => {
+          expect(err.message).to.include('cy.click() failed because this element is detached from the DOM')
+
+          done()
+        })
+
+        cy.get('input:first')
+        .should(($el) => {
```

Comment:
shouldn't go in a `.should` because a `.should` is called multiple times. instead this should be added outside of the `.should` via a `cy.$$` or a `.then`

Reply:
only if it throws. `should` is a convenient way to preserve the subject, but i guess since nothing is returned a `.then` would sufice